docs/decisions/managing-service-dependencies.md

```diff
@@ -43,14 +47,19 @@ If a project anticipates that it could be part of a circular reference then it s
 ```make
 include .env
 
+export
+
 # Set the DEPS_DIR if it hasn't been set already
 ifndef DEPS_DIR
-	DEPS_DIR=${PWD}/deps
+	export DEPS_DIR=${PWD}/deps
 endif
 
 # Space delimited list of dependant projects
 DEPS=project2 project3
 
+# Use override here to ensure this is not inherited from the environment
+override DOCKER_COMPOSE=COMPOSE_PROJECT_NAME=${PROJECT1_PROJECT_NAME} COMPOSE_FILE="${PROJECT1_COMPOSE_FILE}" docker compose
```

Comment:
This will also affect ordinary commands, won't it ?

Reply:
If I understand it well, in off-server we have to define a OFF_SERVER_PROJECT_NAME ?

Reply:
Not yet. It is only a problem when you are working on a project that needs to be pulled in as a dependency as you don't want any environment variables from the depending (parent) project to overwrite your own.